src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -40,6 +40,71 @@ def test_get_error(session):
         assert e.description.find('Attribute is read-only.') != -1
 
 
+def test_commit_after_initiate(session):
```

Comment:
What is this testing? If this is testing that you can call commit after initiate was called in the NI-DCPower API, then that's outside the scope of what should be tested in this file. We aren't testing the NI-DCPower driver, just the Python bindings.